src/you_get/processor/ffmpeg.py

```diff
@@ -92,9 +101,8 @@ def ffmpeg_concat_mp4_to_mpg(files, output='output.mpg'):
     params.append(output + '.mpg')
     params += ['-vcodec', 'copy', '-acodec', 'copy']
     params.append(output)
-    subprocess.call(params)
 
-    if subprocess.call(params) == 0:
+    if subprocess.call(params, stdin=DEVNULL) == 0:
```

Comment:
Here is the dual-call bug I was talking about. I didn't read surrounding code but it's safe to assume that `subprocess.call(params)` shouldn't be called twice in immediate succession.